internal/formats/cyclonedx13json/test-fixtures/snapshot/TestCycloneDxImagePresenter.golden

```diff
@@ -30,13 +40,37 @@
           }
         }
       ],
-      "purl": "a-purl-1"
+      "purl": "a-purl-1",
+      "properties": [
+        {
+          "name": "path",
+          "value": "/somefile-1.txt"
+        },
+        {
+          "name": "layerID",
+          "value": "sha256:d8fa3ce64b8228215c105c9a307ef852b27bf6075cec407ad54509f0a6121c1a"
```

Comment:
Looks like this value should be being redacted per the encoder test changes.

Should value here be `redacted`?

Reply:
Testing it locally I got this layerID but in the CI I got different one. I think the test image that was built is different in my case.

Reply:
I think it's enough to test if the `layerID` is proper `sha256`, because we get it from the `location.FileSystemID` and just add this info to cycloneDX output.